File: fnc/examples/chapter06/rkadapt2.m
%%
f = @(t,u) (t+u).^2;
warning on
[t,u] = rk23(f,[0,1],1,1e-5);
semilogy(t,u)
xlabel('t'), ylabel('u(t)'), title('Finite time blowup')  % ignore this line

File: julia/scripts/chapter06/rkadapt2.jl

f = (u,t) -> (t+u)^2;

include("../FNC.jl")
t,u = FNC.rk23(f,[0,1],1.0,1e-5);

using Plots
plot(t,u,label="",
    xlabel="t",yaxis=(:log10,"u(t)"),title="Finite-time blowup")
